chiselFrontend/src/main/scala/chisel3/core/Bits.scala

```diff
@@ -1020,3 +1021,51 @@ object FixedPoint {
   }
 
 }
+
+/** Data type for representing bidirectional bitvectors of a given width
```

Comment:
In its current incarnation, it really just maps to bidirectional bitvectors in Verilog. If we do generalize it to simply wires in an analog bus, then I do think we should change this description!